ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/ptp-tests-leaf1.cfg

```diff
@@ -47,6 +47,20 @@ vrf instance MGMT
 !
 vrf instance VRF1
 !
+interface Port-Channel1
```

Comment:
I don't see the ptp mpass command here.

Reply:
Havn't added in eos_cli_config_gen, did it now.